src/allmydata/test/test_runner.py

```diff
@@ -252,15 +254,10 @@ class RunNode(common_util.SignalMixin, unittest.TestCase, pollmixin.PollMixin,
               RunBinTahoeMixin):
     """
     exercise "tahoe run" for both introducer, client node, and key-generator,
-    by spawning "tahoe run" (or "tahoe start") as a subprocess. This doesn't
-    get us line-level coverage, but it does a better job of confirming that
-    the user can actually run "./bin/tahoe run" and expect it to work. This
-    verifies that bin/tahoe sets up PYTHONPATH and the like correctly.
-
-    This doesn't work on cygwin (it hangs forever), so we skip this test
-    when we're on cygwin. It is likely that "tahoe start" itself doesn't
-    work on cygwin: twisted seems unable to provide a version of
-    spawnProcess which really works there.
```

Comment:
Curious comment here.  There's no special handling for cygwin in this module, so I guess this must have been stale since 629185d98.

We don't know anything about Cygwin these days, do we?  I guess we will burn that bridge if/when we ever come to it again. :-)

Reply:
Yea I haven't cared about Cygwin for a long time.